main.go

```diff
@@ -216,6 +217,11 @@ func main() {
 				Usage:       "when to build the bpf program. possible options: 'never'/'always'/'if-needed'",
 				Destination: &buildPolicy,
 			},
+			&cli.BoolFlag{
```

Comment:
No need for this new flag - we already have "capture" flag, which can capture "exec/mem/write/all" where we can also add "stack-traces"

Reply:
the capture flag captures artifacts as files into the output directory, wheres this just tells trace to print more info to stdout, not unlike --show-exec-env for example. I don't think it belongs in the capture flag

Reply:
I don't think it's the same. show-exec-env is printing an argument (env) which is part of execve event, that we otherwise hide. Here we want to add stack traces, which will be captured for every event, and that require the use of a special mechanism to capture this data.
I get your point though that this data is part of the printed output and not collected as a file to the tracee output directory, so I don't mind leaving it here

Reply:
Sorry, what's the conclusion here, would you prefer this to be part of the 'capture' flag, or kept as a seperate flag?

Reply:
we've recently agreed to make the capture flag even more about artifact configuration, so I think definitely not under capture. @yanivagman I propose we merge it in with its own flag, and perhaps reconsider later as part of the output flag refactor we plan